app/src/containers/SearchPage/action-types.js

```diff
@@ -3,4 +3,4 @@ export const FETCH_MOVIES_FAILURE = 'FETCH_MOVIES_FAILURE';
 export const FETCH_MOVIES_SUCCESS = 'FETCH_MOVIES_SUCCESS';
```

Comment:
```javascript
export const FETCH_MOVIES_REQUEST: 'FETCH_MOVIES_REQUEST' = 'FETCH_MOVIES_REQUEST';
export const FETCH_MOVIES_FAILURE: 'FETCH_MOVIES_FAILURE' = 'FETCH_MOVIES_FAILURE';
export const FETCH_MOVIES_SUCCESS: 'FETCH_MOVIES_SUCCESS' = 'FETCH_MOVIES_SUCCESS';
export const SET_MOVIES_QUERY: 'SET_MOVIES_QUERY' = 'SET_MOVIES_QUERY';
export const SET_MOVIES_SORT_BY: 'SET_MOVIES_SORT_BY' = 'SET_MOVIES_SORT_BY';
export const SET_MOVIES_SEARCH_BY: 'SET_MOVIES_SEARCH_BY' = 'SET_MOVIES_SEARCH_BY';
```

Reply:
No luck with this, from vcs plugin I got:
`FetchMoviesRequestAction: type FetchMoviesRequestAction = { | +type: any |}`